core/control-plane/control-plane-aggregate-services/src/test/java/org/eclipse/edc/connector/service/protocol/ProtocolTokenValidatorImplTest.java

```diff
@@ -51,19 +53,22 @@ void shouldVerifyToken() {
         when(identityService.verifyJwtToken(any(), any())).thenReturn(Result.success(claimToken));
         when(agentService.createFor(any())).thenReturn(participantAgent);
 
-        var result = validator.verify(tokenRepresentation, "scope", policy);
+        var result = validator.verify(tokenRepresentation, "scope", policy, mock());
```

Comment:
nit: not the biggest fan of mocking value objects. Maybe a `TestRemoteMessage` could be used?

Reply:
Added a `TestMessage`